tests/config_test.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+from honeybee_energy.config import folders
+
+import pytest
+
+
+def test_config_init():
+    """Test the initialization of the config module and basic properties."""
+    assert hasattr(folders, 'energyplus_path')
+    assert folders.energyplus_path is None or isinstance(folders.energyplus_path, str)
```

Comment:
"the real issue" is not very portable, though. I still want the tests to pass if I'm running the tests on a machine with or without EnergyPlus.

Reply:
I'm still not sure what are you trying to test here. What does the value being None or string tells us?

Reply:
It's running the code to make sure that it doesn't fail. If there's no installation found, it will be None. Otherwise it will be a string to the path.

Reply:
I realize that this is not the best example of a test but running the code to make sure there aren't any typos in it still has value.

Reply:
> the code to make sure there aren't any typos in it still has value

I agree with this part of the test: `assert hasattr(folders, 'energyplus_path')`. My comment was on the second part.